src/store/index.js

```diff
@@ -108,7 +108,7 @@ const initialState: StateType = {
 };
 
 const devToolsEnhancer =
-	( 'development' === process.env.NODE_ENV && require( 'remote-redux-devtools' ).default ) ||
+	// ( 'development' === process.env.NODE_ENV && require( 'remote-redux-devtools' ).default ) ||
```

Comment:
If this is not needed, let's remove this line. Otherwise, if this is something you'll need for later, please let's add a `TODO` comment here, explaining what and why succinctly.

Reply:
We don't _need_ it but the `remote-redux-devtools` offers a very nice debugger for Redux. I had to comment it out because it's not super important at the moment and I couldn't make it work in this PR in the short amount of time I was willing to devote to it. I left it in as a comment only as a reminder to try reinstate it in the future. Perhaps marking it with "TODO:" can help there.